### DISCUSSION TITLE
Welcome to intro2sd-claudio_nestor-yanes-mesa-alu0101229942 Discussions!

### DISCUSSION THREAD

**Discussion by claudio4** [ULL-MII-SYTWS-2223/intro2sd-claudio_nestor-yanes-mesa-alu0101229942#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by crguezl** [comment]
**Comentar por comentar algo**  😄

**Reply by crguezl** [reply]
No estoy de acuerdo

**Reply by claudio4** [reply]
prueba
